lib/cosmos/interfaces/protocols/fixed_protocol.rb

```diff
@@ -39,6 +39,9 @@ def initialize(
       @min_id_size = Integer(min_id_size)
       @telemetry = telemetry
       @unknown_raise = ConfigParser::handle_true_false(unknown_raise)
+      @received_time = nil
+      @target_name = 'UNKNOWN'
+      @packet_name = 'UNKNOWN'
```

Comment:
Specs are failing on this because it expects nil. Should this be nil or UNKNOWN?

Reply:
They should be nil.   They probably also need to get set to nil later if an unidentified packet makes it out of identify and finish packet.